diff --git a/base/BUILD.gn b/base/BUILD.gn
index b111dc4c2433f0b62c510d4434a84d3723e85a81..04666c77fcfb03dd83aec3db5ef3f82ccb963dad 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4474,6 +4474,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4512,6 +4513,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("base_cached_flags_java") {
@@ -4538,6 +4540,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/cached_flags/ValuesOverridden.java",
       "android/java/src/org/chromium/base/cached_flags/ValuesReturned.java",
     ]
+    sources += brave_base_cached_flags_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
