diff --git a/build/config/chrome_build.gni b/build/config/chrome_build.gni
index 01e68434921cc62726785f7393c5607c81e7f703..6f2e115c483b53784fa71f737883a5b3d324dbc9 100644
--- a/build/config/chrome_build.gni
+++ b/build/config/chrome_build.gni
@@ -1,6 +1,7 @@
 # Copyright 2015 The Chromium Authors
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+import("//brave/build/config/brave_build.gni")
 
 declare_args() {
   # Select the desired branding flavor. False means normal Chromium branding,
