diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b85f1f44ad06d6cf3696df3c7bb0db97dc936030..b89a90d35984f93ccee3cbdcd079dfe5d492fafe 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2676,6 +2676,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
@@ -6269,6 +6270,7 @@ static_library("browser") {
       "ScreenCaptureKit.framework",  # Available in macOS 12.3, stable in 13.2.
       "UniformTypeIdentifiers.framework",  # macOS 11
     ]
+    public_deps += [ "//brave/browser/mac:keystone_glue" ]
   }
 
   if (is_win || is_linux || is_android || is_chromeos) {
