diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index fc61c9143a9128f231b48f282b9a31966f31da05..8d5626193d43ec8eb45264f37b410dc82a3ce17b 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2674,6 +2674,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
@@ -6266,6 +6267,7 @@ static_library("browser") {
       "ScreenCaptureKit.framework",  # Available in macOS 12.3, stable in 13.2.
       "UniformTypeIdentifiers.framework",  # macOS 11
     ]
+    public_deps += [ "//brave/browser/mac:keystone_glue" ]
   }
 
   if (is_win || is_linux || is_android || is_chromeos) {
