diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index e3556ce88214c6617a247496aa913ac2b7f66f20..df4a45a81faf0e2f835ea911fe356b0cd151ed0d 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -726,6 +726,7 @@ static_library("ui") {
 
     "//chrome/browser/permissions",
   ]
+  public_deps += [ "//brave/browser/ui" ] allow_circular_includes_from += [ "//brave/browser/ui" ]
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -4521,6 +4522,7 @@ static_library("ui") {
       "//third_party/google_toolbox_for_mac",
       "//ui/accelerated_widget_mac:accelerated_widget_mac",
     ]
+    deps += [ "//brave/browser/mac:keystone_glue" ]
     include_dirs = [ "$target_gen_dir" ]
     frameworks = [
       "Carbon.framework",
@@ -4628,10 +4630,13 @@ static_library("ui") {
       ]
       deps += [ "//chrome/updater/app/server/win:updater_legacy_idl" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     if (toolkit_views) {
