diff --git a/chrome/utility/BUILD.gn b/chrome/utility/BUILD.gn
index f0f9123d8699fb6fd173771201abfc048f44310b..6acf3d16802c925ff588c02848340b38fcbd04c4 100644
--- a/chrome/utility/BUILD.gn
+++ b/chrome/utility/BUILD.gn
@@ -72,6 +72,8 @@ static_library("utility") {
     "//ui/base:buildflags",
   ]
 
+  import("//brave/utility/sources.gni") sources += brave_utility_sources deps += brave_utility_deps public_deps += brave_utility_public_deps
+
   if (is_win) {
     deps += [
       "//components/services/quarantine",
