diff --git a/chrome/browser/ui/views/location_bar/location_bar_view.cc b/chrome/browser/ui/views/location_bar/location_bar_view.cc
index c0a9d6fb85c9f671eb30c85fc166623486f4df58..719c3ee60d3b1571af97daf1d7887bb23cafb1b9 100644
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -707,6 +707,7 @@ void LocationBarView::Layout() {
   if (show_overriding_permission_chip)
     text_left += text_overriding_permission_chip_indent;
 
+  BRAVE_LAYOUT_LEADING_DECORATIONS
   LocationBarLayout leading_decorations(LocationBarLayout::Position::kLeftEdge,
                                         text_left);
   LocationBarLayout trailing_decorations(
@@ -786,6 +787,7 @@ void LocationBarView::Layout() {
     }
   };
 
+  BRAVE_LAYOUT_TRAILING_DECORATIONS
   add_trailing_decoration(page_action_icon_container_,
                           /*intra_item_padding=*/0);
   for (ContentSettingImageView* view : base::Reversed(content_setting_views_)) {
