diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 06de47d24d1655549fd39cf930108c85bddeaefc..3b90dba71b786de4ba93f358eb739fb1e48c5e4b 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -367,6 +367,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!ShouldBlockThirdPartyCookies()) {
     return AllowAllCookies{
         ThirdPartyCookieAllowMechanism::kAllowByGlobalSetting};
@@ -464,6 +465,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   const ContentSetting cookie_setting = GetContentSetting(
       url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   const bool is_explicit_setting =
       !setting_info.primary_pattern.MatchesAllHosts() ||
       !setting_info.secondary_pattern.MatchesAllHosts();
