diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 2c68e0606be5e1e89f3b5e95abe4a1feb2cc79b9..a8629443fd20d60e2a2159568c91b1431f4e4994 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -363,6 +363,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
   if (!is_third_party_request) {
     return AllowAllCookies{ThirdPartyCookieAllowMechanism::kNone};
   }
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!ShouldBlockThirdPartyCookies()) {
     return AllowAllCookies{
         ThirdPartyCookieAllowMechanism::kAllowByGlobalSetting};
@@ -460,6 +461,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   const ContentSetting cookie_setting = GetContentSetting(
       url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   const bool is_explicit_setting =
       !setting_info.primary_pattern.MatchesAllHosts() ||
       !setting_info.secondary_pattern.MatchesAllHosts();
