diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index cc90deb8c10b8064183808445d0d9e894d5db323..3fb38a089c9a36707d2f7aa8a396b199c05f3042 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1233,6 +1233,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
