diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index a028b907391ad894adec25c039254506e67408fc..eae6552bc02c23f4b73b1aa6cfc12ad9df3720ba 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2307,6 +2307,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
