diff --git a/third_party/polymer/v3_0/BUILD.gn b/third_party/polymer/v3_0/BUILD.gn
index 51d4c4a4f3b4ef3788af39b5eeabd4bb3d56cfa6..ef9d477a37066c91fa648161c6e13393f9204f7c 100644
--- a/third_party/polymer/v3_0/BUILD.gn
+++ b/third_party/polymer/v3_0/BUILD.gn
@@ -46,6 +46,7 @@ js_files = [
 ]
 
 if (is_chromeos_ash) {
+} if (!is_android) {
   js_files += [
     "iron-dropdown/iron-dropdown.js",
     "iron-dropdown/iron-dropdown-scroll-manager.js",
@@ -176,6 +177,7 @@ ts_library("library") {
   ]
 
   if (is_chromeos_ash) {
+  } if (!is_android) {
     definitions += [
       "components-chromium/iron-dropdown/iron-dropdown.d.ts",
       "components-chromium/iron-dropdown/iron-dropdown-scroll-manager.d.ts",
