diff --git a/tools/typescript/ts_library.py b/tools/typescript/ts_library.py
index cb7b9a22e691aaffe19a810c5aed296bd1039d82..2164dc39efaafb64241566cd3013b6598ca0dbc8 100644
--- a/tools/typescript/ts_library.py
+++ b/tools/typescript/ts_library.py
@@ -280,5 +280,6 @@ def main(argv):
       json.dump(manifest_data, manifest_file)
 
 
+from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/tools/typescript/ts_library.py", globals(), locals())
 if __name__ == '__main__':
   main(sys.argv[1:])
